common/azure-android-client-authentication/build.gradle

```diff
@@ -0,0 +1,117 @@
+buildscript {
```

Comment:
We aren't using gradle to build so we should convert to maven projects during the clean-up.